content/en/agent/faq/macos-agent-run-as-system-service.md

```diff
@@ -0,0 +1,67 @@
+---
+title: How do I set up the Agent to run as a system service on macOS?
+kind: faq
+further_reading:
+- link: "/agent/"
+  tag: "Documentation"
+  text: "Learn more about the Datadog Agent"
+---
+
+On macOS, the Datadog Agent is installed as a user service (for the user that runs the install instructions). This allows the Datadog Agent GUI system tray application to work (while logged in to the macOS GUI as the user that performed the install), but the main drawback is that the Agent only runs when the user that performed the install is logged in using the macOS GUI.
+
+Because of this, by default the Datadog Agent doesn't run in cases where no GUI access to the macOS host is available. Additional steps are therefore required when installing & running the macOS Datadog Agent with no GUI access.
```

Comment:
```suggestion
Because of this, by default the Datadog Agent doesn't run in cases where no GUI access to the macOS host is available. Additional steps are therefore required when installing and running the macOS Datadog Agent with no GUI access.
```